.github/workflows/chaos.yaml

```diff
@@ -11,10 +11,10 @@ jobs:
   pull:
     runs-on: ubuntu-latest
     steps:
-    - name: Set up Go 1.13
+    - name: Set up Go 1.16
       uses: actions/setup-go@v1
       with:
-        go-version: 1.13
+        go-version: 1.16
```

Comment:
we use 1.16.4 in our CI/CD, so I think we can update this to `1.16.4` too.

ref: https://github.com/actions/go-versions/blob/e891add10bdafdc3a5f66def262e646db7d66342/versions-manifest.json#L3